docs/HowTo/Prevent-Slashing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Import or export a slashing protection file
+---
+
+# Slashing Protection
+
+[Teku implements slashing protection] to prevent validators from signing incorrect
+blocks or attestations.
+
+You can import and export the slashing-protection file when migrating validator keys between
+validator clients.
+
+## Importing a slashing-protection file
+
+When importing the slashing-protection file, Teku imports the file to the
+`<data-path>/validators/slashprotection/` directory in the format `<validator-pubkey>.yml`
+(with no 0x prefix).
+
+Teku supports files using the [Minimal] or [Complete] interchange format when importing.
+
+!!! example
+
+    ```bash
+    teku slashing-protection import --data-path=/home/me/me_node --from=/home/slash/validatorNode1.yml
+    ```
+
+In the command line:
+
+* [`--data-path`](../Reference/CLI/CLI-Subcommands.md#data-path) specifies the location of the
+    Teku `data` directory.
+* [`--from`](../Reference/CLI/CLI-Subcommands.md#from) specifies the location of the
+    slashing-protection file.
+
+Teku imports the file to the `/home/me/me_node/data/validators/slashprotection/` directory.
+
+## Exporting a slashing-protection file
+
+Export the slashing-protection file when migrating a validator to a different Teku, or non-Teku
+node. Teku exports the slashing protection file in the [Minimal] format.
+
+!!! example
+
+    ```bash
+    teku slashing-protection export --data-path=/home/me/me_node --to=/home/slash/validatorNode1.yml
```

Comment:
output is json format, and while you could name it a .yml file, that'd be super confusing :)

`--to=slashing-interchange-format-minimal.json`

Reply:
Done